tests/w2w_test.py

```diff
@@ -357,24 +355,28 @@ def test_wrf_remove_urban_output_already_exists_is_overwritten(tmpdir, info_mock
     assert m_time_old != os.path.getmtime(info.dst_nu_file)
 
 
-def test_create_wrf_gridinfo(tmpdir, info_mock):
+def test_get_wrf_grid_info(info_mock):
     info = info_mock(
         {
-            'dst_nu_file': 'testing/5by5.nc',
-            'dst_gridinfo': os.path.join(tmpdir, 'dst_gridinfo.tif'),
+            'dst_file': 'sample_data/geo_em.d04.nc',
         }
     )
-    create_wrf_gridinfo(info)
-    assert os.listdir(tmpdir) == ['dst_gridinfo.tif']
-    tif = rxr.open_rasterio(info.dst_gridinfo)
-    assert tif.rio.crs.to_proj4() == '+init=epsg:4326'
-    assert tif.rio.transform() == Affine(
-        0.01000213623046875,
+    wrf_grid_info = _get_wrf_grid_info(info)
+    assert type(wrf_grid_info) == dict
+    assert list(wrf_grid_info.keys()) == ['crs', 'transform']
```

Comment:
I don't think we need to test this, mypy does all the type checks with the function's type annotation. The keys are already tested with the asserts below, so we can also remove this
```suggestion
```